src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -213,14 +212,6 @@ public void sanitizeForSaving() {
         this.courseId = Sanitizer.sanitizeForHtml(courseId);
         this.giverEmail = Sanitizer.sanitizeForHtml(giverEmail);
 
-        if (recipients != null) {
```

Comment:
Yes, perhaps we will have to do this in another issue to handle sanitization for comments then...